libs/designsystem/menu/src/menu.component.spec.ts

```diff
@@ -286,6 +286,53 @@ describe('MenuComponent', () => {
     });
   });
 
+  describe('interaction with items in condition', () => {
+    it('should select item when clicked when wrapped in if statement', async () => {
+      spectator = createHost(
+        `<kirby-menu>
+          @if (show) {
+            <kirby-item>
+              <p>Action 1</p>
+            </kirby-item>
+          }
+        </kirby-menu>`,
+        { hostProps: { show: false } }
+      );
+      buttonElement = spectator.query('button');
+      spectator.setHostInput({ show: true });
+      await spectator.click(buttonElement);
+      spectator.detectChanges();
+      const ionItem = spectator.query('ion-item');
+      await TestHelper.whenReady(ionItem);
+      const selectableIonItem = ionItem.shadowRoot.querySelector('button'); //"button" only exist if item is selectable
+      expect(selectableIonItem).toBeTruthy();
```

Comment:
Consider readability by using some whitespace to signal AAA pattern:

```suggestion
      buttonElement = spectator.query('button');
      spectator.setHostInput({ show: true });
      
      await spectator.click(buttonElement);
      spectator.detectChanges();
      
      const ionItem = spectator.query('ion-item');
      await TestHelper.whenReady(ionItem);
      const selectableIonItem = ionItem.shadowRoot.querySelector('button'); //"button" only exist if item is selectable
      
      expect(selectableIonItem).toBeTruthy();
```